### DISCUSSION TITLE
General Voltage and first tests

### DISCUSSION THREAD

**Discussion by janczmok1970** [PricelessToolkit/ESPBell-MAX#4] [General]
I would like to have a bit of info, what voltages are needed to trigger it ? and How to test correct operation with a power generator.
As i have a fully charges 18xxx batterie inserted, i still not see the unit lighten up or shows and LEDs. I would assume when the battery is inserted, unit shows some sign on live ?!?

**Comment by PricelessToolkit** [comment]
it will not show signs of life until it is triggered, for testing purposes, you can briefly connect  "ET to battery +" and it will trigger ESPBel-MAX, By default firmware it will stay enabled for 15s max or until the "notification button is pressed.



Triggering voltages are 

"ET" Input 3.3-5v DC Max
"DB" Input Optocoupler ~2-30v AC/DC Max

**Comment by janczmok1970** [comment]
Okay, that triggers it. Need to optimize the setup a bit more. Tomorrow i will test it further with the original bell.
But can you add the yaml files and the jpg for homeassistant to the project ?

**Reply by PricelessToolkit** [reply]
Yep, I will add a .IMG.  and which YAML do you want? Automation?
